ddtrace/tracer/tracer.go

```diff
@@ -128,9 +130,13 @@ func Inject(ctx ddtrace.SpanContext, carrier interface{}) error {
 // payloadQueueSize is the buffer size of the trace channel.
 const payloadQueueSize = 1000
 
-func newUnstartedTracer(opts ...StartOption) *tracer {
+func newUnstartedTracer(opts ...StartOption) (*tracer, *startupInfo) {
 	c := newConfig(opts...)
-	return &tracer{
+	envRules, envRulesErr := samplingRulesFromEnv()
```

Comment:
```suggestion
	envRules, err := samplingRulesFromEnv()
```
Why are we now making error variables this long?

Reply:
Only because the `envRulesErr` variable is used 15 lines below, rather than immediately like `err` usually is.
I can switch it to `err`